topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,182 @@
+
+By linking each mitochondrion’s position across frames, we ensure that intensity fluctuations represent true changes within the same mitochondrial structure, enabling precise analysis of events like mitoflashes.
+
+> <hands-on-title>Tracking Mitochondrial Movement</hands-on-title>
+>
+> 1. {% tool [Perform linking in time series (nearest neighbors)](toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2) %} with the following recommended parameters:
+>    - {% icon param-file %} *"Coordinates (and intensities) of input points"*: Output from the **Spot Detection** tool.
+>    - **Neighborhood size**: Set to `6` pixels as a starting value. Adjust within `1-10` pixels based on the density of points and proximity between spots across frames.
+>    - **Intensity threshold**: A recommended starting value is `25%` of the global maximum intensity. Increase to `30-40%` for stricter filtering if spots have low intensity.
+>    - **Minimum track length**: Use `50%` of the sequence length as a baseline. Adjust based on the expected duration of mitoflash events.
+>    
+>    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Reducing the **Intensity threshold** filters out weaker signals, reducing the number of spurious tracks.
+>
+{: .hands_on}
+
+# Detecting Mitoflashes
+
+Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.
+
+## Step 3: Curve Fitting for Mitoflash Detection
+
+> <hands-on-title>Mitoflash Detection</hands-on-title>
+>
+> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
+>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
+>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
+>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
+>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
+>
+>    This step applies polynomial curve fitting to model intensity fluctuations over time, identifying mitoflash events when intensity values exceed the fitted curve by the given significance level. Key parameters to analyze are **Amplitude** (peak intensity), **Tpk** (time to peak), and **T50** (duration). These values provide initial insights into mitochondrial function, with further interpretation outlined in the following section.
```

Comment:
Not quite clear what the parameters Amplitude, T50, and Tpk refer to. The tool doesn't have those parameters?

Reply:
Thank you for asking! These parameters—**Amplitude**, **Tpk**, and **T50**—are not input parameters of the tool but are derived metrics used to measure mitoflash events. In biology, they are commonly reported to quantify and analyze mitochondrial dynamics. 

For example:
- **Amplitude** measures the peak intensity of the flash.
- **Tpk** reflects the time it takes to reach this peak.
- **T50** indicates the duration of the flash.

These values are useful for plotting and interpreting mitoflash dynamics in biological studies. Perhaps it would be more accurate to call these *"metrics that can be extracted from the output for plotting"*? 

Several papers mention these metrics; for example, [this one](https://pmc.ncbi.nlm.nih.gov/articles/PMC7580095/).

Rather than being annotations, these metrics are quantitative descriptors that help summarize and interpret key features of mitoflash events. Please let me know what you think and if this helps the clarification. I wanted to add this info to guide the user to the analysis and interpretation part.

Reply:
I see, thanks for the clarification! I think this hasn't been sufficiently clear from the tutorial :) I think it would be better to put this into the section after the last step of the workflow, as this seems to be like a description of "further analysis" beyond the workflow. So it would be more natural to first finish the description of the workflow, then describe the further analysis.

> These values are useful for plotting and interpreting mitoflash dynamics in biological studies. Perhaps it would be more accurate to call these "metrics that can be extracted from the output for plotting"?

To further avoid confusion, I think it would be better to avoid the term "parameter" here if possible, "metrics" sounds much better.